feat: Update JavaScript & Node docs to avoid `configureScope()` (#8887)
getsentry/sentry-docs: src/platforms/node/common/enriching-events/scopes/index.mdx

## Configuring the Scope

> There are two main ways to interact with the scope. You can access the current scope via `Sentry.getCurrentScope()`, and use setters on the resulting scope, or you can use the global methods like `Sentry.setTag()` directly, which will under the hood set on the respective scope.

Reviewer: ```suggestion
There are two main ways to interact with the scope. You can access the current scope via `Sentry.getCurrentScope()` and use setters on the resulting scope, or you can use the global methods like `Sentry.setTag()` directly, which will set on the respective scope under the hood.
```

<PlatformContent includePath="enriching-events/scopes/with-scope" />

> The scope inside of the `withScope()` callback is only valid inside of the callback - once the callback ended, the scope will be removed and no longer applied. The inner scope is only applied to events that are captured inside of the callback. `withScope()` will clone (or fork) the current scope, so that the current scope is not modified. This allows you to

Reviewer 2: l: Maybe that's a little clearer, given that "inner scope" isn't mentioned before and people could mistake it for something else

```suggestion
The scope inside of the `withScope()` callback is only valid inside of the callback - once the callback ended, the scope will be removed and no longer applied. The callback scope is only applied to events that are captured inside of the callback. `withScope()` will clone (or fork) the current scope, so that the current scope is not modified. This allows you to
```

> The scope inside of the `withScope()` callback is only valid inside of the callback - once the callback ended, the scope will be removed and no longer applied. The callback scope is only applied to events that are captured inside of the callback. `withScope()` will clone (or fork) the current scope, so that the current scope is not modified. This allows you to

Reviewer: ```suggestion
The scope inside the `withScope()` callback is only valid inside of the callback. Once the callback ends, the scope will be removed and no longer applied. The inner scope is only applied to events that are captured inside of the callback. `withScope()` will clone (or fork) the current scope, so that the current scope is not modified. This allows you to
```
